app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -394,9 +402,6 @@ def process_enrollment_response(enrollment, response)
     else
       handle_unsupported_status(enrollment, response)
     end
-
-    # invoke job to send sms notification
-    send_enrollment_status_sms_notification(enrollment: enrollment)
```

Comment:
I would consider `handle_unsupported_id_type` and `handle_unsupported_secondary_id` to be within the scope of `failed` as defined by the story's original AC, so the user would receive notifications in both of those cases.

Reply:
good catch, thank you. Fixed in f0b14af7b0b6aab291be7a77dea9498818c5cdcf